README.md

```diff
@@ -52,6 +52,17 @@ Getting Started
 ---------------
 The easiest way to run OpenShift Origin is in a Docker container (OpenShift requires Docker 1.6.2 or higher):
 
+You'll need to configure the Docker daemon on your host to trust the Docker registry service you'll be starting.
```

Comment:
This is needed so that this https://github.com/openshift/origin/pull/4263/files#diff-04c6e90faac2675aa89e2176d2eec7d8R90 works.